libyul/backends/wasm/BinaryTransform.cpp

```diff
@@ -26,8 +26,8 @@
 #include <libsolutil/Visitor.h>
 #include <libsolutil/LEB128.h>
 
-#include <boost/range/adaptor/reversed.hpp>
-#include <boost/range/adaptor/map.hpp>
+#include <range/v3/view/reverse.hpp>
+#include <range/v3/view/map.hpp>
 #include <boost/range/adaptor/transformed.hpp>
```

Comment:
Can this be removed?

Reply:
You mean transformed and related code, will do in next commit, In this commit, I will fix map and reverse so that to not have to many fixes in 1 pull request.

Reply:
Done

Reply:
Done